src/Roslyn.Diagnostics.Analyzers/UnitTests/DeclarePublicAPIAnalyzerTests.cs

```diff
@@ -82,11 +82,6 @@ private void VerifyCSharpAdditionalFileFix(string source, string shippedApiText,
             VerifyAdditionalFileFix(LanguageNames.CSharp, source, shippedApiText, oldUnshippedApiText, newUnshippedApiText, onlyFixFirstFixableDiagnostic);
         }
 
-        private void VerifyBasicAdditionalFileFix(string source, string shippedApiText, string oldUnshippedApiText, string newUnshippedApiText, bool onlyFixFirstFixableDiagnostic = false)
```

Comment:
Can you retain this method - though we only have these tests only for C# as of now, I assume we want to add additional file tests even for VB.